_faqs/faq-06.md

```diff
@@ -2,6 +2,18 @@
 question: I have an issue with RIOT code I can't solve. How can I get help?
 ---
 
-Log an issue in GitHub. Then post to the mailing list devel@riot-os.org and/or users@riot-os.org.
-For critical vulnerabilities we would appreciate you to report them with a 90 day heads-up to security@riot-os.org first, before making them publicly available.
-You're also welcome to ask in the IRC channel #riot-os at [freenode.net](http://webchat.freenode.net), but don't be disappointed if everyone there is busy. It is also a very good idea to search our [forum](https://forum.riot-os.org).
+**For critical vulnerabilities we would appreciate you to report them with a 90
+day heads-up to [security@riot-os.org](mailto:security@riot-os.org) first,
+before making them publicly available.**
+
+It's a very good idea to search our
+[forum](https://forum.riot-os.org){:target="_blank"} first, maybe someone has
+already solved your issue! You can also create a post via the forum website or
+via email to [help@riot-os.org](mailto:help@riot-os.org). Another option is to
```

Comment:
A forum account is required to use this email address to submit a new post. This is not immediately clear from this (and I'm not sure if the FAQ offers the space to explain this).

Would it make sense to link to the [forum post explaining how to use it as a mailing list](https://forum.riot-os.org/t/riot-forum-as-a-mailing-list/46) instead.

Reply:
I've modified the sentence to mention that one needs an account and to include a reference to the guide.